packages/core/upload/server/content-types/folder/schema.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+module.exports = {
+  collectionName: 'folders',
```

Comment:
I'm wondering - since this content types will be created once a user installed the Strapi update - do we need some checks/ error handling to make sure, this new content-type doesn't conflict with an existing one? Do we have something in place already?

It would be quite problematic if you can not install a Strapi update because you already have a content type called "folders". 🤔

Reply:
good point, I will update it to upload_folders :)

Reply:
In the next [PR](https://github.com/strapi/strapi/pull/13083#pullrequestreview-939248450) if that's ok for you